dask/dataframe/tests/test_io.py

```diff
@@ -547,11 +547,10 @@ def test_read_csv_of_modified_file_has_different_name():
     with filetext(text) as fn:
         mtime = os.path.getmtime(fn)
         a = read_csv(fn)
+        sleep(1)
         with open(fn, 'a') as f:
             f.write('\nGeorge,700')
             os.fsync(f)
-        while mtime == os.path.getmtime(fn):
-            sleep(0.01)
```

Comment:
So, a few things about this.

1.  We shouldn't budget a whole second of testing time for this one test.  Tests should be very fast.  They're in the inner loop of everyone's development cycle.
2.  We're sleeping at a strange time for this to work, right?  We should be checking this *after* we modify the file.